backend/api/views/tests/test_statistics_r.py

```diff
@@ -263,9 +263,9 @@ def test_get_total(client):
         )
 
     resp = client.get("/api/statistics/total")
-    assert resp.json()["臺南市"]["documents"] == 101
+    assert resp.json()["臺南市"]["documents"] == 9
     count = resp.json()["臺南市"][DocumentDisplayStatusConst.IN_PROGRESS]
-    assert count == 101, f"expect 101 but {count}"
+    assert count == 9, f"expect 9 but {count}"
 
     count = resp.json()["臺北市"][DocumentDisplayStatusConst.IN_PROGRESS]
     assert count == 10, f"expect 10 but {count}"
```

Comment:
Off-topic: it seems the corresponding test data are imported/initialized in the other way. We may pay attention on the approach in case any silent failure in the future.